Sources/PitchSpeller/Data Structures/_Graph.swift

```diff
@@ -59,8 +59,8 @@ struct _Graph<Weight: Weightedness, Pair: SymmetricPair & Directedness> {
         nodes.append(node)
     }
     
-    func insertEdge(from source: Node, to destination: Node, withWeight weight: Weight) {
-        
+    mutating func insertEdge(from source: Node, to destination: Node, withWeight weight: Weight) {
```

Comment:
I like this naming (`withWeight`) better than what I had before. We should really try to keep this style. I've waffled back and forth, but I think it's time to settle down.